packages/chat-core/src/index.ts

```diff
@@ -1079,6 +1114,52 @@ export function createConversation(config: Config): ConversationHandler {
     currentLanguageCode: () => {
       return state.languageCode;
     },
+    getLiveKitCredentials: async () => {
+      const res = await fetch(`${fullApplicationHttpUrl()}/requestToken`, {
+        method: "POST",
+        headers: {
+          ...(config.headers ?? {}),
+          Accept: "application/json",
+          "Content-Type": "application/json",
+          "nlx-conversation-id": state.conversationId,
+          "nlx-sdk-version": packageJson.version,
+        },
+        body: JSON.stringify({
+          languageCode: state.languageCode,
+          conversationId: state.conversationId,
+          userId: state.userId,
+          requestToken: true,
+        }),
+      });
+      if (res.status >= 400) {
+        throw new Error(`Responded with ${res.status}`);
+      }
+      const data = await res.json();
+      if (data?.url == null) {
+        throw new Error("Invalid response");
+      }
+      return data;
+    },
+    terminateLiveKitCall: async () => {
+      const res = await fetch(`${applicationUrl}/terminateVoice`, {
```

Comment:
@peterszerzo  Should be `const res = await fetch(`${fullApplicationHttpUrl()}/terminateVoice`, {` instead (for now)

Reply:
Done!